tests/pages/base.py

```diff
@@ -41,3 +46,18 @@ def languages(self):
         def select_language(self, value):
             el = self.root.find_element(*self._language_locator)
             Select(el).select_by_value(value)
+
+    class FamilyNavigation(PageRegion):
+
+        _tertiary_nav_button = (By.ID, 'fxfamilynav-tertiarynav-trigger')
+        _tertiary_nav_menu = (By.ID, 'fxfamilynav-tertiarynav')
+
+        def click_tertiary_menu(self):
+            self.selenium.find_element(*self._tertiary_nav_button).click()
+            Wait(self.selenium, self.timeout).until(
+                expected.visibility_of_element_located(self._tertiary_nav_menu))
+
+        @property
+        def tertiary_menu_visible(self):
+            el = self.selenium.find_element(*self._tertiary_nav_menu)
+            return el.is_displayed()
```

Comment:
If we create a `pages.firefox` module, and this menu only applies to those pages, we could move it into `pages/firefox/base.py`, make that extend `pages/base.py` and make the appropriate pages extend the new `FirefoxBasePage`. That said, I don't feel strongly about this - it's a better organisational structure if appropriate but adds an additional layer.